composer.json

```diff
@@ -73,7 +73,8 @@
         "phpunit/phpunit": "^9.5",
         "symfony/browser-kit": "5.4.*",
         "symfony/css-selector": "5.4.*",
-        "symfony/phpunit-bridge": "5.4.*"
+        "symfony/phpunit-bridge": "5.4.*",
+        "phpstan/phpdoc-parser": "^1.33"
```

Comment:
This is not a dev dependency.

Reply:
Also, it would be better to add version >=2.0 to conflicts section, instead of requring the specific lower version.